FindAIETools.cmake

```diff
@@ -147,6 +147,24 @@ foreach(comp ${AIETools_FIND_COMPONENTS})
 		set(aieVersionSpecificPath "unknown")
 	endif()
 
+	if(${comp} STREQUAL "AIE")
+		set(aieVersionTargetPath "target")
+	elseif(${comp} STREQUAL "AIE2")
+		set(aieVersionTargetPath "target_aie_ml")
+	else()
+		string(TOLOWER ${comp} aiearch)
+		set(aieVersionTargetPath "target_${aiearch}")
+	endif()
+
+	# Find chesscc
+	find_path(AIETOOLS_${comp}_TARGET_DIR "chesscc" PATHS ${AIETOOLS_DIR}/tps/lnx64/${aieVersionTargetPath}/bin/LNa64bin
```

Comment:
Does this generate the path or do you need to take the basename?

Reply:
this generates a path based on the AIETOOLS_DIR that was found before.

Reply:
`-- Found AIE2 target dir: /scratch/jmelber/vitis-aie-essentials/tps/lnx64/target_aie_ml/bin/LNa64bin`